docs/src/onlinereso.rst

```diff
@@ -0,0 +1,55 @@
+.. _librarywebsites:
+On-line resources
+=================
+
+i-PI resources
+~~~~~~~~~~~~~~
+
+For more information about i-PI and to download the source code go to
+http://ipi-code.org/.
+
+In http://gle4md.org/ one can also obtain colored-noise parameters to
+run Path Integral with Generalized Langevin Equation thermostat
+(PI+GLE/PIGLET) calculations.
+
+Python resources
+~~~~~~~~~~~~~~~~
+
+For help with Python programming, see
+`www.python.org <www.python.org>`__. For information about the NumPy
+mathematical library, see `www.numpy.org <www.numpy.org>`__, and for
+worked examples of its capabilities see
+`www.scipy.org/Tentative_NumPy_Tutorial <www.scipy.org/Tentative_NumPy_Tutorial>`__.
+Finally, see http://hgomersall.github.io/pyFFTW/ for documentation on
+the Python FFTW library that is currently implemented with i-PI.
+
+
+Client code resources
+~~~~~~~~~~~~~~~~~~~~~
+
+Several codes provide out-of-the-box an i-PI interface, including 
+ASE, 
+CASTEP, 
+CP2K,
+DFTB+,
+elphmod,
+ffsGDML,
+FHI-aims, 
+Lammps, 
+librascal, 
+Quantum ESPRESSO, 
+Siesta,
+VASP, Yaff, VASP.
```

Comment:
VASP is here twice, and I am not sure the VASP interface is shipped with VASP. Could someone confirm? Also LAMMPS write themselves with all capitals.

Reply:
I took out i-pi since VASP, since as far I can tell it doesn't appear in their documentation. 
If someone who knows wants to add a small section on how to patch VASP, due to its large userbase it would be great

Reply:
Hey Yair, in the README there is description on how to patch VASP with i-PI 

examples/clients/vasp